libs/lz4/src/main/java/org/elasticsearch/lz4/ESLZ4Compressor.java

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
Yeah, we need to fix the license header here. We should remove the Elastic header and instead add `@notice` to the first line of the Apache 2 header to indicate this is vendored code. Just do a search on that string to see other examples. Same applies to all files in this project.